documentation/content/tests.rst

```diff
@@ -21,5 +27,25 @@ Or run individual tests with::
 
   ./run_tests.sh test TESTFILE1 ...
 
+To limit the scope of the tests, pass the flag `--unit` or `--integration` as
+the first argument to any of the examples above. So, the syntax is::
+
+  ./run_tests.sh [scope] [command] [args]
+
+Where `[scope]` can be `--unit` or `--integration`. The placeholder `[command]`
+can be either `list`, `test` or simply omited in order to run all tests.
+
+The integration tests can take over 10 minutes to run in the first time  because
+podman is building the container images to be used in the tests, which  requires
+installing kw's dependencies in the supported distros.  After  the  images  have
+been built and cached, running the integration tests  should  take  only  a  few
+seconds each time.
+
+Then, the local kw repo is copied to the containers and installed  again,  which
+takes very few seconds. For optimization purposes,  the  containers  are  reused
+accross tests. If you add a new commit or checkout to another branch, such  that
+HEAD points to another commit, the containers  will  be  destroyed  and  created
+again in order to install the current local version of kw.
+
```

Comment:
Add one example that shows how to run all integration tests and another one that shows how to run one specific test.